src/backend/gl/src/device.rs

```diff
@@ -396,13 +405,14 @@ impl Device {
         desc_remap_data: &mut n::DescRemapData,
         name_binding_map: &mut FastHashMap<String, (n::BindingTypes, pso::DescriptorBinding)>,
     ) -> n::Shader {
-        assert_eq!(point.entry, "main");
         match *point.module {
             n::ShaderModule::Raw(raw) => {
+                assert_eq!(point.entry, "main");
```

Comment:
FWIW we probably don't need these because we appear to be passing `point.entry` through to naga/spirv-cross

Reply:
Ok, I'll remove these in the follow-up.